apps/wing/src/wingc.ts

```diff
@@ -24,19 +24,27 @@ export interface WingCompilerLoadOptions {
 }
 
 export async function load(options: WingCompilerLoadOptions) {
+  const preopens = {
+    // .jsii access
+    [WINGSDK_MANIFEST_ROOT]: WINGSDK_MANIFEST_ROOT,
+    ...(options.preopens ?? {}),
+  } as Record<string, string>;
+
+  if (process.platform === "win32") {
+    for (const [key, value] of Object.entries(preopens)) {
+      delete preopens[key];
+      preopens[normalPath(value)] = value;
+    }
```

Comment:
Mutating the same collection you're iterating over.... living life dangerously I see? 😄

Reply:
😎 
Technically iterating over `Object.entries(preopens)`, so should be completely safe to mutate the original afaik